kamailio/trunks/config/kamailio.cfg

```diff
@@ -1693,6 +1697,28 @@ route[CONTROL_MAXCALLS] {
     }
 }
 
+# This route sets $var(is_within_country) to 1 for within country calls
+route[IS_WITHIN_COUNTRY] {
+    $var(is_within_country) = 0;
+
+    sql_query("cb", "SELECT countryCode FROM Companies C JOIN Countries CS ON C.countryId=CS.id WHERE C.id=$dlg_var(companyId)", "ra");
+
+    if ($dbr(ra=>rows) == 0) {
+        return;
```

Comment:
Is `sql_result_free("ra")` required before this return?

Reply:
Solved, thanks